system-test/src/main/java/org/apache/mesos/elasticsearch/systemtest/Main.java

```diff
@@ -22,7 +20,7 @@ public static void main(String[] args) throws InterruptedException {
         MesosCluster cluster = new MesosCluster(
             MesosClusterConfig.builder()
                 .mesosImageTag(MESOS_IMAGE_TAG)
-                .slaveResources(new String[]{"ports(*):[9200-9200,9300-9300]", "ports(*):[9201-9201,9301-9301]", "ports(*):[9202-9202,9302-9302]"})
+                .slaveResources(new String[]{"ports(*):[4000-4000,5000-5000]", "ports(*):[4001-4001,5001-5001]", "ports(*):[4002-4002,5002-5002]"})
```

Comment:
I will revert back. No reason. I was just testing that the docker ports expose worked with different ports and forgot to move them back.